src/components/website-management/website-form-components/customization/customization-components/CookieBanner.js

```diff
@@ -0,0 +1,303 @@
+import React, { useContext, useEffect, useState } from "react";
+import Typography from "@mui/material/Typography";
+import CustSelectBox from "./CustSelectBox";
+import { Box } from "@mui/system";
+import { Button } from "@mui/material";
+import { BannerContext } from "./BannerContext";
+import CustSelectBoxBanner from "./CustSelectBoxBanner";
+import axios from "axios";
+import Snackbar from "@mui/material/Snackbar";
+import MuiAlert from "@mui/material/Alert";
+import CookieSettingsDialog from "./CookieSettingDialog";
+
+const CookieBannner = () => {
+  const { setColor, setTop, setLeft, setChosenCont, color } =
+    useContext(BannerContext);
+
+  const [templates, settemplates] = useState([]);
+
+  const [banner, setBanner] = useState([]);
+
+  const [selectedTemplate, setSelectedTemplate] = useState({});
+
+  const [alignmentValue, setAlignmentValue] = useState("");
+
+  const [positionValue, setPositionValue] = useState("");
+
+  const [colorValue, setColorValue] = useState("");
+
+  const passedId = useContext(BannerContext).websiteId;
+
+  /*Sanck bar useStates*/
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
+
+  const [snackbarMessage, setSnackbarMessage] = useState("");
+
+  const [error, setError] = useState(false);
+
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const [showDialog, setShowDialog] = useState(false);
```

Comment:
Let's group related code together. Eg: let's put context statements together and usestate statements together as separate groups.

Reply:
fixed